app/public/css/styles.css

```diff
@@ -1429,3 +1429,66 @@ p {
   background-position: center;
   background-image: url('../assets/alert_icon_warning.svg');
 }
+
+/* MAIN PAGE */
+
+.page-wrapper {
+  display: flex;
+  height: 100%;
+  width: var(--width);
+  margin: 0 auto;
+}
+
+.page-main {
+  flex: 0 0 66%;
```

Comment:
This can be replaced with `width: 66%`. It's more readable and you'll have the same result.

Reply:
Will do